### ISSUE TITLE
The syntax for spatial predicates is difference than that for other predicates.  Why?

### ISSUE THREAD

**Issue by pvretano** [opengeospatial/ogcapi-features#493]
The syntax for most predicates in CQL is "expression OPERATOR expression".  The syntax for the spatial operators, however, is OPERATOR(expression,expression).  This seem oddly incongruous and was inherited from CSW which probably inherited it from OGC 06-103r4.  OGC 01-103r4 was probably using a functional notation (i.e. OPERATOR(expression,expression)=0|1) to accommodate the limitations (i.e. no Boolean type) of RDBMSs at the time.

So, is this something we should change? 

I should point out that unlike 06-103r4, the requirement in CQL is that OPERATOR(spatialExpr,spatialExpr) evaluates to TRUE or FALSE so someone writing a CQL filter does not need to say **filter=INTERSECTS(geom,POLYGON(...)=1**.

**Comment by cportele** [comment]
In general, I agree. I did not raise an issue about this so far, because of the CQL legacy and the fact that this is what existing CQL implementations support, but going forward it may be hard to explain to others that never heard about CSW. I think we have two options:

* Leave it as it is for now, but indicate in a note that the current syntax `INTERSECTS(x,y)` is only kept for backwards compatibility and we would prefer to change this to `x INTERSECTS y` before finalizing the document and that we would welcome comments.
* The other way round - change it, highlight the change and ask for comments (now or in the public review).

I have a slight preference for the first option, mainly because this is what current OGC API Features implementations support at this point in time (although it should not be difficult to update them to the other style) and we want to create a draft.1 tag next week. 

We should discuss and decide what we do for the draft release next week in the SWG meeting on Monday. Comments are welcome.

**Comment by pvretano** [comment]
@cportele agreed, discuss Monday.  My going in position is to rip the bandaid off now! :)

**Comment by aaime** [comment]
I agree the current syntax is odd... but it's also well known and documented:

- CQL has been used in CSW for a long time
- Shortly after, GeoServer adopted it as an alternative filtering language for WFS and WMS
- A few years down the road, GeoServer devs created an extended version of CQL called ECQL

If you go for a clean break, may I recommend to give the language a different name, or at least use a different version number of sorts, so that it's clear it's not "your grandpa's CQL"? e.g. CQL.2021 for example.

**Comment by cportele** [comment]
Meeting 2020-12-21: We will leave the notation for now as it is, but @cportele will add notes to the document about the two aspects (different notation, different return value compared to Simple Features) and invites comments that we will review after the public review.

**Comment by philvarner** [comment]
My perspective is that most clients for this API will be using a client SDK to construct queries rather than writing them directly in text or JSON. These clients can construct different (possibly *radically different*) ways of constructing queries that work best for their language paradigm (e.g., tightly-typed like the "sttp" Scala library, or Pythonic like the Requests library, though I'm sure there are better examples from SQL libraries for those languages). As long as CQL meets the requirements for expressiveness and isn't too onerous to mechanically generate, I don't think it matters if it's not as clean or beautiful as it could be, and there's a lot of value in it already having had a lot of use.

**Comment by cportele** [comment]
Meeting 2021-06-14: Given that we have decided to include breaking changes with the old CQL, we should consider to harmonize the style and use functions for the temporal and array operators, too. See also https://github.com/opengeospatial/ogcapi-features/pull/587#issuecomment-860850178 on the use of the standard comparison operators for date/date-time properties.
